sysapi/sysapi_util/Unmarshal_TPMU_CAPABILITIES.c

```diff
@@ -87,6 +87,11 @@ void Unmarshal_TPMU_CAPABILITIES(
 			Unmarshal_TPML_ECC_CURVE( sysContext, &capabilities->eccCurves );
 			break;
 #endif
+#ifdef TPM_CAP_VENDOR_PROPERTY
```

Comment:
it is: https://github.com/01org/tpm2-tss/blob/master/include/sapi/tss2_tpm2_types.h#L264